source/educators/how-tos/update_rerun_course.rst

```diff
@@ -97,3 +97,13 @@ for launch, see :ref:`Launch`.
 
```

Comment:
Above in this file there is an `.. only:: Partners` stanza. These need to all be found and deleted.

Reply:
This file is under aulusneo's cleanup, but we had to touch it to change the name of a target. It'll get cleaned under aulusneo's ongoing work.